ui/imports/shared/popups/keycard/helpers/KeyPairUnknownItem.qml

```diff
@@ -0,0 +1,170 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import QtQml.Models 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1 as StatusQUtils
+import StatusQ.Components 0.1
+import StatusQ.Controls 0.1
+
+import utils 1.0
+import shared.stores 1.0 as SharedStore
+
+Rectangle {
+    id: root
+
+    property string keyPairPubKey: ""
+    property string keyPairName: ""
+    property string keyPairIcon: ""
+    property string keyPairImage: ""
+    property string keyPairDerivedFrom: ""
+    property string keyPairAccounts: ""
+
+    color: Style.current.grey
+    radius: Style.current.halfPadding
+    clip: true
```

Comment:
Why clipping?

Reply:
copy/paste or was trying something during development and forgot about it